src/ucs/datastruct/bitmap.h

```diff
@@ -568,6 +592,75 @@ _UCS_BITMAP_DECLARE_TYPE(128)
 _UCS_BITMAP_DECLARE_TYPE(256)
 
 
+/* Helper function to set all bitmap bits to 0 */
+static UCS_F_ALWAYS_INLINE void
+ucs_bitmap_bits_clear(ucs_bitmap_word_t *bits, size_t num_words)
```

Comment:
Maybe `ucs_bitmap_clear`?
Removing `_bits` saves redundant chars. and doesn't introduce any ambiguity.

Reply:
wanted to distinguish between functions that operated on "bits, num_words" from functions that operate on a bitmap struct. the ones with "_bits" are helper functions